src/notebook.c

```diff
@@ -473,6 +473,12 @@ static void show_tab_bar_popup_menu(GdkEventButton *event, GeanyDocument *doc)
 	g_signal_connect(menu_item, "activate", G_CALLBACK(on_close_other_documents1_activate), doc);
 	gtk_widget_set_sensitive(GTK_WIDGET(menu_item), (doc != NULL));
 
+	menu_item = ui_image_menu_item_new(GTK_STOCK_CLOSE, _("Close Documents To The Right"));
```

Comment:
That's a good point. Also, we generally use sentence case, not title case right?